readthedocs/projects/static-src/projects/js/import.js

```diff
@@ -282,6 +282,9 @@ function ProjectImportView(instance, config) {
             .then(function (data) {
                 self.get_organizations();
                 self.get_accounts();
+                // This will trigger a new
+                // request to re-sync the repos
+                self.filter_by(self.filter_by());
```

Comment:
I believe the observable should have a `notifySubscribers` method that we can call, instead of resetting the variable like this to notify subscribing observables.

Reply:
I'll try to take a deeper look to the docs and see what I can find.